hw/drivers/adc/gpadc_da1469x/src/gpadc_da1469x.c

```diff
@@ -30,6 +30,10 @@
 
 static struct da1469x_gpadc_dev *da1469x_gpadc_dev;
 
+// 1/0.180265 * 1024 = 5680, 1/0.06907 * 1024 = 14825, 1/0.065067 * 1024 = 15737
+// 1/0.068603 * 1024 = 14926, 1/0.066215 * 1024 = 15464, 1/0.170288 * 1024 = 6013
+static uint16_t ildo_scalar[] = {5680,14825,15737,14926,15464,6013};
```

Comment:
You can make it a const array since it never gets modified, also, we generally have spaces after `,`s inside the array. That would get the line beyond 79 chars, so you should try to wrap it, that is again part of our coding style.

Reply:
I didn't make it const since each scalar value will be trimmed on a per unit basis and stored in OTP. So, they will be pulled during initialization.